src/routes/console/project-[project]/auth/security/updatePasswordPersonalData.svelte

```diff
@@ -0,0 +1,58 @@
+<script lang="ts">
+    import { invalidate } from '$app/navigation';
+    import { Submit, trackError, trackEvent } from '$lib/actions/analytics';
+    import { CardGrid, Heading } from '$lib/components';
+    import { Dependencies } from '$lib/constants';
+    import { Button, Form, InputSwitch } from '$lib/elements/forms';
+    import FormList from '$lib/elements/forms/formList.svelte';
```

Comment:
lets use the named import

Reply:
Hmm the formList.svelte file does not export any named components ?

Reply:
but `$lib/elements/forms` does

Reply:
Updated! Thanks 👌